src/components/pages/Login/Login.js

```diff
@@ -1,77 +1,80 @@
 import React from 'react';
-import { Button } from 'semantic-ui-react';
 import { useState } from 'react';
 import { Redirect } from 'react-router-dom';
 import {
   loginHandler,
-  getToken
+  getUser
 } from '../../../utils/handlers/authenticationHandlers';
 import PropTypes from 'prop-types';
+import LoginForm from './LoginForm';
+import validateLoginForm from './loginValidation';
 
-function Login({ setUser }) {
-  const [credentials, changeCredentials] = useState({
+function Login({ setUser, ...props}) {
+  const [state, setState] = useState({
     username: '',
-    password: ''
+    password: '',
+    errors: {},
+    loginIn: false
```

Comment:
I know that `loginIn` is about saying if it's currently in the process of login in order to display the Loading... indicator. But at the beginning I thought this was about if it was successfully logged in. Maybe just 'loading' would work better and make it easier to see on the first read.

Reply:
Oh, I see this PR is only about the responsive. Forget it, I'll put it on Pavel's!